store/src/java/com/zimbra/cs/service/account/AccountDocumentHandler.java

```diff
@@ -50,34 +50,7 @@ protected Element proxyIfNecessary(Element request, Map<String, Object> context)
             throw e;
         }
     }
-    
-    /*
-     * bug 27389
-     */
-    protected boolean checkPasswordSecurity(Map<String, Object> context) throws ServiceException {
```

Comment:
why is this removed?

Reply:
To reuse this method, we have moved it to the parent class ie DocumentHandler. It will be available to inherited classes like AccountDocumentHandler or AdminDocumentHadler.